graylog2-server/src/main/java/org/graylog2/alarmcallbacks/AlarmCallbackConfigurationServiceMJImpl.java

```diff
@@ -94,4 +112,9 @@ private AlarmCallbackConfigurationAVImpl implOrFail(AlarmCallbackConfiguration c
             throw new IllegalArgumentException("Supplied output must be of implementation type AlarmCallbackConfigurationAVImpl, not " + callback.getClass());
         }
     }
+
+    private static class CountAggrType {
```

Comment:
Please create a proper class (e. g. with `@AutoValue`) with `hashCode()` and `equals()`, especially since instances of this class are stored in a `HashMap`.